prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -24,6 +33,7 @@ class DatabaseDeliveryAccess(private val db: DatabaseAccess = BaseEngine.databas
      */
     override fun <T> fetchActions(
         organization: String,
+        orgSuffix: String?,
```

Comment:
This should probably be named 'orgService' since that is what we call it elsewhere.

Reply:
done!